config/config-example.json

```diff
@@ -1,38 +1,34 @@
-{
```

Comment:
Instead of updating this example config file, could we document this new handler in the README instead?

Reply:
I've added the handler now in the Readme similar to other handlers and reverted the changes made in config example json